main.py

```diff
@@ -29,7 +31,8 @@ def show_frame(self, page_name):
 
 
 if __name__ == "__main__":
-    app = SampleApp()
+    app = App()
     app.title("Video Compressor Application")
     app.geometry("950x560")
+    app.resizable(False, False)
```

Comment:
Is this intentional, design-wise? What happens if the user try to resize the window?

Reply:
yes, i made it into fixed-size window so that it won't be a problem with GUI later future